alcs-frontend/src/app/services/application/application-document/application-document.service.spec.ts

```diff
@@ -51,11 +67,27 @@ describe('ApplicationDocumentService', () => {
     expect(res[0].uuid).toEqual('1');
   });
 
+  it('should make a get call for getApplicantDocuments', async () => {
+    httpClient.get.mockReturnValue(
+      of([
+        {
+          uuid: '1',
+        },
+      ]),
+    );
+
+    const res = await service.getApplicantDocuments('1');
+
+    expect(httpClient.get).toHaveBeenCalledTimes(1);
+    expect(res.length).toEqual(1);
+    expect(res[0].uuid).toEqual('1');
```

Comment:
The description only says a GET request is made, it doesn't say it needs to return a result or that it must contain a UUID. Are these really necessary?